### ISSUE TITLE
Problem with sortable directive

### ISSUE THREAD

**Issue by YuriyPobezhymov** [angular-ui/angular-ui-OLDREPO#255]
Hello.
I had a pb with defining custom events and attributes from controller for sortable element.

```
function Ctrl($scope) {
 $scope.$on('$viewContentLoaded', function () {
    $scope.sortableOptions = {
      "delay": 250,
      "update": function(e, ui) {
        console.log("update");
     }
  }
}
```

I found example of date directive using and then I looked indepth of angular-ui.js.
My code start working after I did change line 1040 to:
`opts = angular.extend({}, options, scope.$eval(attrs.uiSortable));`

Also I don't understand pb, when I use ng-model for my list. It's enables events that defined in angular-ui.js, but those events never called (update event directy) and my custom event doesn't called (I did debug with firebug and it never step in function opts.update on line 1062).

Thanks, Yuriy.

**Comment by c0bra** [comment]
Yes there's something definitely wrong with the sortable directive and how it's handling the options coming in. It seems like they're being scrubbed out on line 1040 as you say. I don't think "attrs.uiOptions" is even defined.

**Comment by lstrzelecki** [comment]
We tested this solution, and after change in sources sortable started working. It is unacceptable for changing sorces in library. So could you release fixed version, as soon as possible?

**Comment by ajoslin** [comment]
Would you mind making a pull request with the fix?

**Comment by lstrzelecki** [comment]
Done

**Comment by ProLoser** [comment]
Landed in 9163ffec2be63cabeed703b674f4c1f654c85e1f #291
